tendrl/common/alert.py

```diff
@@ -0,0 +1,153 @@
+import json
+from tendrl.common.config import TendrlConfig
+from tendrl.common.etcdobj.etcdobj import Server as etcd_server
+from tendrl.common.singleton import to_singleton
+
+
+alert_severity_map = {
+    'INFO': 0,
+    'WARNING': 1,
+    'CRITICAL': 2
+}
+config = TendrlConfig("commons", "/etc/tendrl/tendrl.conf")
+
+
+class Alert(object):
+
+    def __init__(self, alert_id, node_id, time_stamp, resource, current_value,
+                 tags, alert_type, severity, significance, ackedby, acked,
+                 pid, source):
+        self.alert_id = alert_id
+        self.node_id = node_id
+        self.time_stamp = time_stamp
+        self.resource = resource
+        self.current_value = current_value
+        self.tags = tags
+        self.alert_type = alert_type
+        self.severity = severity
+        self.significance = significance
+        self.ackedby = ackedby
+        self.acked = acked
+        self.pid = pid
+        self.source = source
+
+    def to_json_string(self):
+        return json.dumps(self.__dict__)
+
+
+@to_singleton
+class AlertUtils(object):
+    def __init__(self):
+        etcd_kwargs = {
```

Comment:
I feel config should come from AlertManager and we should not refer create a local etcd_client. If possible pass from AlertManager.